Cargo.toml

```diff
@@ -21,3 +21,6 @@ members = [
   "tokio-udp",
   "tokio-uds",
 ]
+
+[patch.crates-io]
+tokio-executor = { path = "tokio-executor" }
```

Comment:
Prevent sub-crates that do not depend on the change from pulling in `tokio-executor` from crates.io.